ext/quic/01_quic.js

```diff
@@ -0,0 +1,246 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+import { core, primordials } from "ext:core/mod.js";
+const {
+  op_quic_listen,
+  op_quic_accept,
+  op_quic_connect,
+  op_quic_accept_bi,
+  op_quic_accept_uni,
+  op_quic_open_bi,
+  op_quic_open_uni,
+  op_quic_send_datagram,
+  op_quic_read_datagram,
+  op_quic_close_endpoint,
+  op_quic_close_connection,
+  op_quic_connection_closed,
+  op_quic_max_datagram_size,
+  op_quic_get_send_stream_priority,
+  op_quic_set_send_stream_priority,
+  op_quic_get_conn_remote_addr,
+} = core.ensureFastOps();
+const {
+  Uint8Array,
+  TypedArrayPrototypeSubarray,
+  SymbolAsyncIterator,
+} = primordials;
+import {
+  getWritableStreamResourceBacking,
+  ReadableStream,
+  readableStreamForRid,
+  WritableStream,
+  writableStreamForRid,
+} from "ext:deno_web/06_streams.js";
+
+class QuicSendStream extends WritableStream {
```

Comment:
In other APIs we usually hoist these out of the readable/writable stream (ie: TcpConn has the `setNoDelay` and `setKeepAlive` functions.

Reply:
there can be many of these per quic connection. an alternative could be `QuicSendStream { writable, sendOrder }` but this matches the WebTransport api better and requires fewer objects.

Reply:
That's a good point, thanks.